src/shared/compatibility.js

```diff
@@ -883,3 +883,35 @@ PDFJS.compatibilityChecked = true;
 })();
 
 }
+
+// Provides a partial workaround when blend modes are not supported
+// See issue https://github.com/mozilla/pdf.js/issues/5264
+// NOTE: This is not a blend mode polyfill, but a way to see blended content
+// that would otherwise be unviewable
+// Support: IE11
+(function checkBlendModeSupport() {
```

Comment:
Also, this code won't be bundled into `GENERIC` builds as-is, it would need to be placed *before* the `URL` polyfill above.